docs/introduction.md

```diff
@@ -33,35 +33,48 @@ class.
-thought of in a hierarchical way.
+A _Partial Ancestral Graph_ (_PAG_) describes an equivalence class of _MAGs_. _PAGs_ may also contain "wild-card" or
+"circle" edges (`-o`), which can either be a directed or undirected arrow head, i.e. `A -o B` can be  resolved to
+`A -- B` or `A -> B`. The `o` end is referred to as "unknown" in this package.
 
-![DAG CPDAG PAG](images/DAG_CPDAG_PAG.png)
+| Type of Graph                      | DAG                | CPDAG                | MAG                | PAG                |
+|:-----------------------------------|:------------------:|:--------------------:|:------------------:|:------------------:|
+| Tester method                      | `graph.is_dag()`   |         :x:          |        :x:         |         :x:        |
+| Direct edges `->`                  | :white_check_mark: |  :white_check_mark:  | :white_check_mark: | :white_check_mark: |
+| Undirected edges `--`              |        :x:         |  :white_check_mark:  | :white_check_mark: | :white_check_mark: |
+| Latent confounder edges `<>`       |        :x:         |         :x:          | :white_check_mark: | :white_check_mark: |
+| Wildcard edges `o-`, `o>` and `oo` |        :x:         |         :x:          |        :x:         | :white_check_mark: |
 
+See `cai_causal_graph.type_definitions.EdgeType` for all the supported edge types in `cai_causal_graph`.
 Note that the `cai_causal_graph.causal_graph.CausalGraph` class can contain all the aforementioned edge types, and 
-can therefore represent _DAGs_, _CPDAGs_, _MAGs_, and _PAGs_.
+can therefore represent the entire hierarchy of _DAGs_, _CPDAGs_, _MAGs_, and _PAGs_.
+
+:::info
+Discovering a single _DAG_ for a given data set is difficult. Certain causal relationships are indistinguishable from
+each other with only observational data, because they encode the same conditional independencies between variables. 
+The set of such causal relationships is called the _Markov equivalence class_ (_MEC_) for a particular set of nodes.
+
+Multiple _DAGs_/_CPDAGs_/_MAGs_/_PAGs_ can be consistent with the same _MEC_. For instance, if you identify the
+graphical structure `X -> Y -> Z`, then corresponding data would show that `X` **is** independent of `Z` given `Y`.
+However, the graphical structures `X <- Y <- Z` and `X <- Y -> Z` would lead to the exact same conditional independence
+test result as above. Only if the graphical structure found was a collider connection `X -> Y <- Z` would you be able to
+identify the structure from observational data, because the data would tell you that `X` and `Z` are independent, but
+become dependent given `Y`.
+:::
+
+:::warning
+In a `CPDAG` the `--` implies an existence of an edge which can be in either direction, `<-` or `->`. In a `PAG`
+the `--` that is a possible outcome of a wildcard edge (for example `o-`) can also resolve to no edge et all.
```

Comment:
Not sure about this. How is it no edge? That would maybe happen in the `DAGSampler` but not in this package.